src/vs/workbench/parts/snippets/electron-browser/snippetsFile.ts

```diff
@@ -232,7 +232,11 @@ export class SnippetFile {
 			body = body.join('\n');
 		}
 
-		if ((typeof prefix !== 'string' && !Array.isArray(prefix)) || typeof body !== 'string') {
+		if (Array.isArray(description)) {
+			description = description.join('\n');
+		}
+
+		if ((typeof prefix !== 'string' && !Array.isArray(prefix)) || typeof body !== 'string' || typeof description !== 'string') {
```

Comment:
This makes the `description` property mandatory and therefore breaks existing snippets with one.

Reply:
@jrieken So, what has to be further?

Reply:
:+1: